WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -53,23 +53,23 @@ private extension ProductFormViewController {
         navigationItem.rightBarButtonItems = [UIBarButtonItem(title: updateTitle, style: .done, target: self, action: #selector(updateProduct))]
 
         if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.editProductsRelease2) {
-            navigationItem.rightBarButtonItems?.insert(moreBarButton(), at: 0)
+            navigationItem.rightBarButtonItems?.insert(moreOptionsBarButton(), at: 0)
         }
         removeNavigationBackBarButtonText()
     }
 
-    func moreBarButton() -> UIBarButtonItem {
+    func moreOptionsBarButton() -> UIBarButtonItem {
```

Comment:
The name sounds more like a property in Swift-land. May I suggest to name it as what it truly does — `createMoreOptionsBarButtonItem()`?